warmup.py

```diff
@@ -225,7 +227,16 @@ class RadialDistribution:
 
     """
     def __init__(self, rmax, dr, rmin=0.0):
-        pass
+        self.rmax = rmax
+        self.rmin = rmin
+        r_range = self.rmax - self.rmin
+        num_bins = np.round(r_range/dr)
+        self.dr = r_range/num_bins
+        self.edges = np.arange(rmin, rmax + self.dr/2, self.dr)
+        self.centers = 0.5*(self.edges[:-1] + self.edges[1:])
+        self._deltaV = (4/3)*np.pi*(self.edges[1:]**3 - self.edges[:-1]**3)
+        self._rdf = np.zeros_like(self.centers)
```

Comment:
Since you implement this in `self.reset()`, you might consider calling that rather than duplicating the code.